azuread/resource_group.go

```diff
@@ -103,3 +172,40 @@ func resourceGroupDelete(d *schema.ResourceData, meta interface{}) error {
 
 	return nil
 }
+
+func fetchAllMembers(groupId string, client graphrbac.GroupsClient, ctx context.Context) ([]string, error) {
+	it, err := client.GetGroupMembersComplete(ctx, groupId)
+
+	if err != nil {
+		return nil, fmt.Errorf("Error listing existing group members from Azure AD Group with ID %q: %+v", groupId, err)
+	}
+
+	existingMembers := make([]string, 0)
+
+	for it.NotDone() {
+		currUser, _ := it.Value().AsUser()
```

Comment:
We should probably support all types here. That way should be able to reuse this function for the group_member object